models/config/coupons_model.php

```diff
@@ -174,13 +174,21 @@
             'label' => 'lang:igniter.coupons::default.label_validity',
             'type' => 'radiotoggle',
             'default' => 'forever',
+            'span' => 'left',
             'options' => [
                 'forever' => 'lang:igniter.coupons::default.text_forever',
                 'fixed' => 'lang:igniter.coupons::default.text_fixed',
                 'period' => 'lang:igniter.coupons::default.text_period',
                 'recurring' => 'lang:igniter.coupons::default.text_recurring',
             ],
         ],
+        'apply_automatically' => [
```

Comment:
you missed this, also check the coupons_model $casts attributes and maybe rename the lang string to match?

Please do another test to confirm all works as expected so I can merge immediately after ;)

Reply:
Changed and tested.